views/components/LineChart/UPlotMultiLineChart.tsx

```diff
@@ -19,27 +19,45 @@ export default class UPlotMultiLineChartComponent extends React.Component<
   IUPlotMultiLineChartProps,
   IUPlotMultiLineChartState
 > {
+  fmtDate = uPlot.fmtDate("{YYYY}-{MM}-{DD}-{h}:{mm}:{ss}{aa}");
+  localTz = new Intl.DateTimeFormat().resolvedOptions().timeZone;
+  tzDate = ts => uPlot.tzDate(new Date(ts * 1e3), this.localTz);
```

Comment:
Could we declare these varialbes `fmtDate`, `localTz`, and `tzDate` outside the class? Let's have them all as a `const`.